Codeinwp/neve
# Height of primary menu items

### Summary

All menu items are wrapped in an **a** tag. By giving it height: 100%, align-items will center them in the flex container, on the same axis/height with the rest of the menu items.

### Will affect visual aspect of the product

NO

### Screenshots 
![neve-polylang](https://user-images.githubusercontent.com/26867648/50162508-3ae07400-02e7-11e9-963f-1805fc68a610.png)

### Test instructions


- Create a menu and assign it to the primary location
- Install Polylang plugin. 
- Add the language switcher to the menu
- Use the flags only, no text, no drop-down for the language switcher
- Check the vertical alignment


Closes #279
